Sensors/SMCBatteryManager/BatteryManager.cpp

```diff
@@ -29,14 +29,16 @@ void BatteryInfo::validateData() {
 	}
 
 	if (designCapacity < state.lastFullChargeCapacity) {
-		SYSLOG("binfo", "battery reports lower design capacity than maximum charged (%u/%u)",
+		SYSLOG("binfo", "battery %d reports lower design capacity than maximum charged (%u/%u)", id,
 			   designCapacity, state.lastFullChargeCapacity);
 		if (state.lastFullChargeCapacity < ValueMax) {
 			auto temp = designCapacity;
 			designCapacity = state.lastFullChargeCapacity;
 			state.lastFullChargeCapacity = temp;
 		}
 	}
+
+	SYSLOG("binfo", "battery %d cycle count %d remaining capacity %ld", id, cycle, state.lastFullChargeCapacity);
```

Comment:
Should we really always print it? Maybe it is better for debugging log?

Reply:
I'm not sure if it's a common interests on how battery capacity degrades with more charging cycles. For example, mine got 88% of designed capacity with 230 cycles and I'm little worried after fixing _BIX, but I can't found some log to trace that. However, maybe it's not a realistic solution as logs may be wiped out.

And I'm sorry for a force push or an amend after the pull, because I want to keep a clean commit log. That part fix an always true statement by making it persistent in batteryInfo.